README.md

```diff
@@ -200,18 +200,45 @@ The tool discovers checks first, then appends any `additional_checks` are concat
 
 ### Excluding specific jobs
 
-The easiest way to exclude a single job is via the `name` parameter:
+You can exclude specific jobs by using their names:
```

Comment:
For killing the word "easiest" 👍🏻 

Otherwise nice additions with examples to the different options.